packages/ffe-form-react/src/PhoneNumber.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {string, bool, func} from 'prop-types';
+import uuid from 'uuid';
+
+import i18n from './i18n/i18n';
+
+export default function PhoneNumber({
+                                        number,
+                                        countryCode,
+                                        onCountryCodeChange,
+                                        onNumberChange,
+                                        onCountryCodeBlur,
+                                        onNumberBlur,
+                                        locale,
+                                        disabled,
+                                        countryCodeInvalid,
+                                        numberInvalid,
+                                        className
+                                    }) {
+
+    const supportedLocales = ['nb', 'nn', 'en'];
+    const text = i18n[supportedLocales.indexOf(locale) !== -1 ? locale : 'nb'];
+
+    const numberId = uuid.v4();
+    const countryCodeId = uuid.v4();
```

Comment:
Here you'll get different ids every time you render, I'm not sure if screen readers appreciate that? It shouldn't be a problem for browser users.

Reply:
Well, we've pulled these out of the render method in a few other components, so it would probably be the "cleanest" thing to do.

That being said, you have two choices here:
1 - verify that this doesn't piss off the most common screen readers
2 - rewrite the component to a class component, and initialize the IDs as class fields.

I'd suggest approach number 2 ;-)